CONTRIBUTING.md

```diff
@@ -353,3 +354,17 @@ If you're unsure where to add code (renderer, UI, extensions, plugins) see the b
 Podman Desktop is moduralized into extensions for each 'Provider'. You can also create and add your own extension.
 
 See our [EXTENSIONS.md](/EXTENSIONS.md) document for more details.
+
+## Maintainer tasks
+
+List of maintainer tasks to help the project run smoothly.
+
+### Triaging
+
+New issues will have the [triage](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage) label assigned. As a maintainer, you'll have to curate these issues into the appropriate [area labels](https://github.com/containers/podman-desktop/labels).
+
+Ideally complete the task every morning:
+
+1. [View all the current triage labeled issues.](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage)
+2. Add the appropriate [area label](https://github.com/containers/podman-desktop/labels) that matches the issue. You can use "bulk edit" by clicking the checkmark of the issues and using the label dropdown selection.
```

Comment:
```suggestion
2. Add the appropriate [area label](https://github.com/containers/podman-desktop/labels?q=area%2F) that matches the issue. You can use "bulk edit" by clicking the checkmark of the issues and using the label dropdown selection.
```

Reply:
We have 39 labels! For someone triaging for the first time it might be overwhelming.

Can we consider simplifying this list?

For docs, when we design steps in a procedure, we try (sometimes hard) to stick to the "rule of seven", as it is the maximum a human brain is supposed to be able to process in one go. Getting closer to 7 could be beneficial here too.

We could:

- Keep the 7 label are used for more than 20 issues.
- Remove the 12 labels are unused or used for one unique issue.
- Try to regroup the remaining labels.
- Add a description to the labels, with the triage person in mind.